pom.xml

```diff
@@ -10,6 +10,8 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+
+        <logback.version>1.2.9</logback.version>
```

Comment:
Add version for each dependency.
It can lead to versions conflict, if you will share them.

Reply:
fix